scripts/west_commands/sbom/README.rst

```diff
@@ -0,0 +1,361 @@
+  * Exclamation mark ``!`` to exclude files.
+
+  For example, if you want to include all :file:`.c` files from the current directory and all subdirectories recursively:
+
+  .. code-block:: bash
+
+     --input-files '**/*.c'
+
+  Make sure to have correct quotes around globs, to not have the glob resolved by your shell, and go untouched to the command.
+
+  You can prefix a pattern with the exclamation mark ``!`` to exclude some files.
+  Patterns are evaluated from left to right, so ``!`` excludes files from patterns before it, but not after.
+  For example, if you want to include all :file:`.c` files from the current directory and all subdirectories recursively except all :file:`main.c` files, run:
+
+  .. code-block:: bash
+
+     --input-files '**/*.c' '!**/main.c'
+
+* You can read a list of input files from a file:
+
+  .. parsed-literal::
+     :class: highlight
+
+     --input-list-file *list_file*
+
+  It does the same as ``--input-files``, but it reads files and patterns from a file (one file or pattern per line).
+  Files and patterns contained in the list file are relative to the list file location (not the current directory).
+  Comments starting with a ``#`` character are allowed.
+
+
+.. _west_sbom Specifying output:
+
+Specifying output
+=================
+
+You can specify the format of the report output using the ``output`` argument.
+
+* To generate a report in HTML format:
+
+  .. parsed-literal::
+     :class: highlight
+
+     --output-html *file-name.html*
+
+  :ref:`west_sbom HTML report overview` provides more details about the report.
+
+  If you use ``-d`` option, you do not need to specify any output argument.
+  The :file:`sbom_report.html` file is generated in your build directory
+  (the first one if you specify more than one build directory).
+
+* To generate a cache database:
+
+  .. parsed-literal::
+     :class: highlight
+
+     --output-cache-database *cache-database.json*
+
+  For details, see ``cache-database`` detector.
+
+.. _west_sbom Detectors:
+
+Detectors
+=========
+
+The ``ncs-sbom`` command has the following detectors implemented:
+
+* ``spdx-tag`` - search for the ``SPDX-License-Identifier`` in the source code or the binary file.
+  For guidelines, see `SPDX identifier`_. Enabled by default.
+
+* ``full-text`` - compare the contents of the source file with a small database of reference texts.
+  The database is part of the ``ncs-sbom`` command. Enabled by default.
+
+* ``scancode-toolkit`` - license detection by the `Scancode-Toolkit`_. Enabled and optional by default.
+
+  If the ``scancode`` command is not on your ``PATH``, you can use the ``--scancode`` option to provide it, for example:
+
+  .. code-block:: bash
+
+     --scancode ~/scancode-toolkit/scancode
+
+  This detector is optional because is significantly slower than the others.
+
+* ``external-file`` - search for license information in an external file. Enabled by default.
+
+  The external file has the following properties:
```

Comment:
```suggestion
  This detector has the following properties:
```

Reply:
Not `detector`, but the external file that is used by this detector. The `external file` was mentioned in above sentence, so I think it is pretty clear. I would not change this.